### DISCUSSION TITLE
How to add HTML attribute to header in Bard

### DISCUSSION THREAD

**Discussion by podcastdecontent** [statamic/cms#5012] [Q&A]
We want to realize a TOC (table of content) with site internal hyperlinks. For that we need headings to have custom IDs to link to them.

Is there a way (for a editor) to add HTML attributes to headings (and other HTML tags)?

**Comment by duncanmcclean** [comment]
Not sure if this is helpful at all but maybe you could take a look at the ['Toc for Bard & Markdown'](https://statamic.com/addons/goldnead/toc-for-bard-and-markdown) addon to see how it does it (or even just use it instead of building it yourself 😅 )
